src/collectd_dogstatsd.py

```diff
@@ -122,6 +122,11 @@ class SignalfxPointSender(object):
     def __init__(self, config, log):
         self.config = config
         self.log = log
+
```

Comment:
I would actually prefer create a set_host() method on the dogstatsd instance and a method that sets both places.